packages/core/__tests__/ci-env.config.spec.ts

```diff
@@ -14,18 +14,19 @@ import {
   TravisCI,
   UnsupportedCI,
 } from '../src/ci-providers';
-import { CIEnvConfig } from '../src/config/ci-env.config';
+import { isCi, detectCi } from '../src/config/ci-env.config';
```

Comment:
I guess it would be better to use capital letters for `CI` instead of `Ci`. Just reading `Ci` seems a bit odd.

```suggestion
import { isCI, detectCI } from '../src/config/ci-env.config';
```

Reply:
Feel free to ignore this if you like or if you see that linter complains about using big letters in `CI` word.